wordpress.org/public_html/wp-content/themes/pub/wporg-support/sass/style-original.scss

```diff
@@ -119,7 +119,7 @@ ul#views {
 .bbp-reply-header .bbp-meta .bbp-reply-permalink {
 	float: none;
 	margin-left: 0;
-	color: $color__link;
+	color: #0073aa;
```

Comment:
Sass color vars have been removed, so reverting this to what I believe would be the original value